Packs/SumoLogic_Cloud_SIEM/Integrations/SumoLogicCloudSIEM/SumoLogicCloudSIEM.py

```diff
@@ -649,49 +674,62 @@ def fetch_incidents(client: Client, max_results: int, last_run: Dict[str, int],
     # Each incident is a dict with a string as a key
     incidents: List[Dict[str, Any]] = []
 
-    q = 'timestamp:>={}'.format(latest_created_time)
+    # set query values that do not change with pagination
+    q = 'created:>={}'.format(insight_timestamp_to_created_format(latest_created_time))
```

Comment:
can you elaborate on the difference between `timestamp` and `created`, other than the pagination-related metadata? would like to understand if we're changing the behavior in a way we're breaking backward compatibility

Reply:
Yep, `timestamp` is derived from the latest Signal that's part of the Insight. `created` is the time the Insight was created.

We have a feature that allows users to create Insights from historical Signals. To make sure those manually created Insights are included when continuously fetching Insights to Demisto we're changing to use `created`.

Insight `created` comes after Signal creation so we're not going to miss any Insights when making this change, and since we're tracking Insight IDs with `last_fetch_ids` we should not get any duplicates either.